backend/ethereum/channel/asset.go

```diff
@@ -32,7 +32,28 @@ import (
 )
 
 // Asset is an Ethereum asset.
-type Asset = wallet.Address
```

Comment:
How about defining it as `type Asset walllet.Address` ?

The advantage in this case would be that, we wouldn't need `NewAssetFromAddress` and `EthAddress` methods.
The previously used ways to covert to and from wallet.Address or common.Address will continue to work.

Reply:
It's possible, but in this case methods like `MarshalBinary` wouldn't be inherited and therefore we would have to reimplement them. I don't see why `NewAssetFromAddress` and `EthAddress` then. How would you create a new Asset with your approach? How would you get the underlying `common.Address`. The options I can see appear quite lengthy.

Reply:
On how to get the EthAddress, the approach would be what it was before the change `ethwallet.Address(Asset)`.  But, as you have pointed out, it is a bit verbose to use.

However, since the `MarshalBinary` function wouldn't be inherited, the current approach seems better.